bibliopixel/util/enum.py

```diff
@@ -0,0 +1,11 @@
+from enum import IntEnum
+
+
+def resolve_enum(enum_type, val):
```

Comment:
def resolve_enum(enum_type, value):
        try:
            return enum_type[value]  # It's a string.
        except:
            return enum_type(value)   # It's an int.

https://docs.python.org/3.4/library/enum.html#programmatic-access-to-enumeration-members-and-their-attributes

Yeah, I know I wrote the original one, but I was wrong originally.  :-D